src/lib/utilities/register-commands.ts

```diff
@@ -0,0 +1,12 @@
+import { envIsDefined, envParseString } from '@skyra/env-utilities';
+import { Registry } from '@skyra/http-framework';
+
+export async function registerCommands() {
+	const registry = new Registry({});
+
+	if (envIsDefined('REGISTRY_GUILD_ID')) {
+		await registry.registerAllCommandsInGuild(envParseString('REGISTRY_GUILD_ID'));
+	} else {
+		await registry.registerGlobalCommands();
+	}
```

Comment:
Throw if it isnt defined. We never want to register Ring commands globally.

Reply:
It's fine, for `/history` at least.